### PR TITLE
JBPM-9216: Upgrade of JDBC postgresql driver to 42.2.14

### PR DESCRIPTION
**Pull request by mareknovotny**
https://issues.redhat.com/browse/JBPM-9216
Merge together with https://github.com/kiegroup/appformer/pull/1020

Unifiying on one version of postgresql while upgrading

### CODE REVIEW

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

**Comment by mareknovotny** [comment]
@Ginxo some weird error on github actions yaml file `2020-08-05T16:16:35.4360162Z [ERROR]  EISDIR: illegal operation on a directory, read`

**Comment by mareknovotny** [comment]
all failures are on functional jbpm quartz or timer services related tests.

**Comment by Ginxo** [comment]
> @Ginxo some weird error on github actions yaml file `2020-08-05T16:16:35.4360162Z [ERROR] EISDIR: illegal operation on a directory, read`

This is due the pull_request.yml definition is taken from `mareknovotny:JBPM-9216` which is an old definition. These kind of problems will be solved by https://issues.redhat.com/browse/BXMSPROD-924

**Comment by mareknovotny** [comment]
@gmunozfe what about those 80 failures in FDB?

**Comment by gmunozfe** [comment]
> @gmunozfe what about those 80 failures in FDB?

@mareknovotny they're unrelated to the content of this PR, as Quartz db for these tests is h2, not postgresql. They need a deeper study because I've seen them from time to time, but IMHO they shouldn't block this.

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -4400,12 +4399,6 @@
         <version>${version.org.yaml.snakeyaml}</version>
       </dependency>
 
-      <dependency>
```
Comment (elguardian) [inline pom.xml:4403]: why are you removing the dependency ?

Comment (mareknovotny) [reply pom.xml:4403]: there were 2 postgresql dependencies, one older with `postgresql` groupId and the newer which is latest one with `org.postgresql` groupId

**Comment by mareknovotny** [comment]
hmm FDB ended with Timeout has been exceeded

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

**Pull request merged**